content/200-concepts/050-overview/300-prisma-in-your-stack/01-rest.md

```diff
@@ -141,11 +141,25 @@ app.delete(`/post/:id`, async (req, res) => {
 
 ## Ready-to-run example projects
 
-You can find several ready-to-run examples that show how to implement a REST API with Prisma Client in the [`prisma-examples`](https://github.com/prisma/prisma-examples/) repository.
+You can find several ready-to-run examples that show how to implement a REST API with Prisma Client, as well as building full applications, in the [`prisma-examples`](https://github.com/prisma/prisma-examples/) repository.
```

Comment:
I'm not a native speaker but somehow using the gerund here sounds a bit weird to me 🤔 if this sentence is grammatically proper and sounds good in your ears, then just ignore this comment 😅